pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -59,14 +59,14 @@ in
 , benchmarkDepends ? [], benchmarkHaskellDepends ? [], benchmarkSystemDepends ? [], benchmarkFrameworkDepends ? []
 , testTarget ? ""
```

Comment:
btw. why don‘t you change other lines, like this one, too?

Reply:
There isn't corresponding lines at the end for them like there were for the ones I changed
```
// optionalAttrs (checkPhase != "")     { inherit checkPhase; }
```
that ties their evaluation to that of the keys of the attribute set.

That is, as it was, you can pass an error expression for them and still be able to access basic derivation values like `name`. This wasn't true for the ones I changed. With the change it now is also.